### PR TITLE
Added asset managment using flask-assets

### PR DESCRIPTION
**Pull request by arychj**

### CODE REVIEW

**Review by raetha — APPROVED** [review]
Looks reasonable to me. Only thing I see is that it appears there are things in requirements.txt that aren't in use yet, but guessing that is for future expansion.

**Comment by arychj** [comment]
@raetha I just froze the pip environment into `requirements.txt`, the packages that aren't being used are actually dependencies of the other dependencies.

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# bibliopompica
\ No newline at end of file
+# bibliopompica
+
+Build assets: `npm run build`
+
+Run test server: `npm run server`
```
Comment (zeebok) [inline README.md:5]: So we don't need to interact with Flask/Python?

Comment (zeebok) [reply README.md:5]: OH I see in the json... guess I should've kept reading before commenting :P

**Review by zeebok — APPROVED** [review]
Looking good!

File: README.md (hunk 1 above)
Comment (arychj) [reply README.md:5]: I kept forgetting the environment variable for flask :p

**Pull request merged**